common/src/files.rs

```diff
@@ -259,4 +259,70 @@ mod test {
         let id_map = vec![s1, s2, s3];
         sort_stringify_id_map(&id_map, true);
     }
+
```

Comment:
How are these tempfiles deleteed?

Reply:
NamedTempFile relies on its destructor to do cleaning.

Reply:
> NamedTempFile relies on its destructor to do cleaning.

NamedTempFile relies on its destructor to do cleaning.